pkgs/tools/text/kenlm/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, cmake, boost, eigen, xz, zlib, bzip2 }:
+
+stdenv.mkDerivation rec {
+  pname = "kenlm-unstable";
+  version = "2020-11-03";
```

Comment:
```suggestion
  pname = "kenlm";
  version = "unstable-2020-11-03";
```

Reply:
@IvarWithoutBones suggested the opposite of this change and referred to #68518